tools/validators/instance_validator/build/lib/tests/generate_universe_test.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google LLC
```

Comment:
did you create this code? or is this a mistaken part of the PR. 

if it is truly new code, we should split the PR to be an ontology update and a code update. 

(there appear to be many code updates in this PR, this comment applies to all of them)